supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -34,7 +34,7 @@
 [discrete]
 [[we-suggest]]
 ==== image:images/no.png[no] we suggest (verb)
-*Description*: Do not use "we suggest". Use a more direct construction or use "recommend". For example, instead of "We suggest that you make a backup of your data disk", write "Back up your data disk", or "It is recommended that you back up your data disk".
+*Description*: Do not use "we suggest". Use a more direct construction instead. For example, instead of "We suggest that you make a backup of your data disk", write "Back up your data disk".
```

Comment:
Replace the phrasal verb "back up" with archive?

Reply:
"Back up" is fine per ISG and SSG. Any particular reason to change the wording?

Reply:
A bit of minimalism. Usually, phrasal verbs should be avoided, where possible. I thought I read somewhere about these verb phrases with translations. Maybe, I misread this.

Reply:
I'm only removing the recommended wording, not necessarily trying to reword the rest of the content. I'd probably prefer to just keep as is